parsons/airtable/airtable.py

```diff
@@ -84,7 +84,7 @@ def get_records(self, fields=None, max_records=None, view=None, formula=None, so
         """
 
         # Raises an error if sort is None type. Thus, only adding if populated.
-        kwargs = {'fields': fields, 'max_records': max_records, 'formula': formula}
+        kwargs = {'fields': fields, 'max_records': max_records, 'view': view, 'formula': formula}
```

Comment:
What is this from?

Reply:
This was from another branch. Reverted it!